setup.cfg

```diff
@@ -48,7 +48,7 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
-    ignore::RuntimeWarning:desc.compute
+    ignore::RuntimeWarning
```

Comment:
Are there new warnings to ignore?

Reply:
we shouldn't ignore all runtime warnings unless there is a good reason we can't avoid them

Reply:
Meant to revert this, thanks for catching. It was warnings I would get from the scipy root solver but we dont get those anyomre now